src/data-table/data-table-column.js

```diff
@@ -19,6 +21,8 @@ const defaultProps = {
   format: null,
   template: null,
   nullTemplate: null,
+  sortable: false,
```

Comment:
maybe set `sortable` to default true here otherwise it would be a breaking change, don't forget to update readme